Tests/SwiftDocCTests/Test Bundles/AvailabilityOverrideBundle.docc/ComplexAvailable.md

```diff
@@ -0,0 +1,11 @@
+# Complex Available Article
```

Comment:
Is this being used in the test? I'm only seeing `MyClass` being tested above – not a huge deal if we include this as well, just curious if maybe a test case got lost along the way.

Reply:
Nope, it's not being used, just ended up being copied in with the symbol graph. I'll remove it since it doesn't really need to be there.